test/tests/functional/rs_mocks/add_remove_tests.js

```diff
@@ -603,6 +603,7 @@ describe('ReplSet Add Remove (mocks)', function() {
             done();
           }, 1000);
         }, 500);
+        // }, 650);
```

Comment:
The add_remove_tests that are failing pass when I change the timeout to 650ms. My current implementation is slowing down the replset connections. Is there something in my implementation that could be slowing this down?